src/index.html

```diff
@@ -11,7 +11,195 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+
+      <fieldset class="fieldset1">
+        <legend>Personal information</legend>
+        
+        <div class="firstInput">
+          <label>
+            Surname:
+            <input 
+            type="text"
+            name="surname"
+            autocomplete="off"
+            >
```

Comment:
```suggestion
            <input 
              type="text"
              name="surname"
              autocomplete="off"
            >
```
take a look over [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) and fix it in all places
<img width="1038" alt="Screenshot 2022-12-01 at 11 18 16" src="https://user-images.githubusercontent.com/94289038/205021687-b827b192-75e5-4309-ae5b-8dc202cabee0.png">